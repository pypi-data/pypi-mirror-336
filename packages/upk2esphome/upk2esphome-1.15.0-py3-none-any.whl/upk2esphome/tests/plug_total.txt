{"manufacturer":"Enbrighten","name":"WFD4105IndoorMiniSmartPlug","key":"keyjup78v54myhan","ap_ssid":"enbrighten","github_issues":[],"image_urls":["enbrighten-wfd4105-indoor-mini-smart-plug.jpg"],"profiles":["oem-bk7231n-plug-1.1.8-sdk-2.3.1-40.00"],"schemas":{"000003a1gg":[{"type":"obj","mode":"rw","property":{"type":"bool"},"id":1},{"type":"obj","mode":"rw","property":{"min":0,"max":86400,"scale":0,"step":1,"type":"value"},"id":9},{"type":"obj","mode":"rw","property":{"range":["off","on","memory"],"type":"enum"},"id":38},{"type":"obj","mode":"rw","property":{"range":["relay","pos","none","on"],"type":"enum"},"id":40}]},"device_configuration":{"bt_type":0,"ch_cddpid1":9,"ch_dpid1":1,"ch_num":1,"crc":17,"jv":"1.0.0","led1_lv":0,"led1_pin":6,"module":"CB2S","net_trig":4,"netled1_lv":0,"netled1_pin":7,"netled_reuse":1,"reset_t":5,"rl1_lv":1,"rl1_pin":24,"total_bt_lv":0,"total_bt_pin":26,"total_stat":2}}
