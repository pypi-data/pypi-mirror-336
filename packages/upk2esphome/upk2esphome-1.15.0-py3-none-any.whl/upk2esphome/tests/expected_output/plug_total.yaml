switch:
  - platform: gpio
    id: switch_1
    name: Relay 1
    pin: P24
    on_turn_on:
      - light.turn_on: light_status
    on_turn_off:
      - light.turn_off: light_status

binary_sensor:
  - platform: gpio
    id: binary_switch_all
    pin:
      number: P26
      inverted: true
      mode: INPUT_PULLUP
    on_press:
      then:
        - switch.toggle: switch_1

light:
  - platform: status_led
    id: light_status
    pin:
      number: P7
      inverted: true
